platform/ios/ios.xcodeproj/project.pbxproj

```diff
@@ -1724,6 +1730,8 @@
 				DA8848491CBAFB9800AB86E3 /* MGLMapboxEvents.m */,
 				9620BB361E69FE1700705A1D /* MGLSDKUpdateChecker.h */,
 				9620BB371E69FE1700705A1D /* MGLSDKUpdateChecker.mm */,
+				AC518DFD201BB55A00EBC820 /* MGLConfig.h */,
+				AC518DFE201BB55A00EBC820 /* MGLConfig.m */,
```

Comment:
Do we intend to expand this class beyond telemetry/metrics? If not, I think it’d be useful to give it a more descriptive name, perhaps by inserting a noun — `MGLTelemetryConfig`, etc.